test/interop/C/c-backend/use_testing.lastcompopts

```diff
@@ -1 +0,0 @@
--Llib/ -ltesting `$CHPL_HOME/util/config/compileline --libraries`
```

Comment:
It looks like you've removed a lot of .lastcompopts but I still see some that got moved over, was that intentional? (I can't comment on them directly, but there's one four files down from here)

Reply:
Many of the `.lastcompopts` in LLVM has a trailing `-ltesting` at the end, in such cases I added that to the version of the test in the common path (as I felt it couldn't hurt).